main/reference/zoe-api/zoe.md

```diff
@@ -314,6 +314,7 @@ key:value pairs:
   by its **timer** and **deadline** properties.
   **timer** must be a timer, and **deadline** must be a [BigInt](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt) value interpreted with respect to the timer.
   Some example timers use Unix epoch time, while others count block height.
+  The proposer cannot exit the seat before the deadline, but the contract can exit the seat early on their behalf.
```

Comment:
```suggestion
  The proposer cannot exit the seat before the deadline - the seat is in the `waived` state until the deadline, but the contract can exit the seat early on the proposer's behalf.
```

Reply:
Realizing the section this is proposing to edit is no longer there. Will update this to target the https://docs.agoric.com/reference/zoe-api/zoe.html#proposals section.